packages/jaeger-ui-components/src/TraceTimelineViewer/SpanDetail/AccordianText.test.js

```diff
@@ -33,24 +33,26 @@ describe('<AccordianText>', () => {
   };
 
   beforeEach(() => {
-    wrapper = mount(<AccordianText {...props} />);
+    const r = render(<AccordianText {...props} />);
+    rerender = r.rerender;
   });
 
   it('renders without exploding', () => {
-    expect(wrapper).toBeDefined();
-    expect(wrapper.exists()).toBe(true);
+    expect(screen).toBeDefined();
   });
 
   it('renders the label', () => {
-    const header = wrapper.find(`[data-test-id="AccordianText--header"] > strong`);
-    expect(header.length).toBe(1);
-    expect(header.text()).toBe(props.label);
+    const { getByText } = within(screen.getByTestId('AccordianText--header'));
+    expect(getByText(props.label)).toBeInTheDocument();
   });
 
   it('renders the content when it is expanded', () => {
-    wrapper.setProps({ isOpen: true });
-    const content = wrapper.find(TextList);
-    expect(content.length).toBe(1);
-    expect(content.prop('data')).toBe(warnings);
+    props.isOpen = true;
+    rerender(<AccordianText {...props} />);
+    expect(screen.getAllByTestId('TextList')).toHaveLength(1);
+    const { getByText } = within(screen.getByTestId('TextList'));
+    warnings.forEach((warning) => {
+      expect(getByText(warning)).toBeInTheDocument();
+    });
```

Comment:
let's just keep it simple:

```suggestion
    props.isOpen = true;
    render(<AccordianText {...props} />);
    warnings.forEach((warning) => {
      expect(screen.getByText(warning)).toBeInTheDocument();
    });
```